docs/contribution.md

```
@@ -0,0 +1,43 @@
+## 1. Prerequisites
+- Sync your branch with `develop`.
+
+## 2. Branch Naming
+Branches should tell you what they do at a glance:
+This is only a suggestion:
+```
+<type>-<short-description>
+```
+Examples: `feat-add-instructor-domain`, `fix-login-bug`
+
+## 3. Commit Messages
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)